client/state/action-types.js

```diff
@@ -21,6 +23,8 @@ export const COMMENTS_REQUEST_SUCCESS = 'COMMENTS_REQUEST_SUCCESS';
 export const COMMENTS_REQUEST_FAILURE = 'COMMENTS_REQUEST_FAILURE';
 export const COMMENTS_UNLIKE = 'COMMENTS_UNLIKE';
 export const CURRENT_USER_ID_SET = 'CURRENT_USER_ID_SET';
+export const CUSTOMIZER_CLOSED = 'CUSTOMIZER_CLOSED';
```

Comment:
Are these behaviors planned to migrated to Redux in the near future? Not sure I like the idea of mixing Flux action types into `state/action-types.js`.